bundles/org.openhab.binding.folderwatcher/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -6,5 +6,6 @@
 	<type>binding</type>
 	<name>FolderWatcher Binding</name>
 	<description>This binding will monitor specified location for new files and trigger event channel with new file names.</description>
+	<connection>none</connection>
```

Comment:
I believe since #14669 this might be:
```suggestion
	<connection>hybrid</connection>
```

Even before that it was probably `local` because of FTP support.

Reply:
Fixed, thanks!